packages/instant/src/components/ui/flex.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { ColorOption, styled } from '../../style/theme';
+import { cssRuleIfExists } from '../../style/util';
+
+export interface FlexProps {
+    direction?: 'row' | 'column';
```

Comment:
Just a thought:  instead of explicitly defining the possible values for these properties, we could get them from https://github.com/frenic/csstype which `React.CSSProperties` uses internally:

![image](https://user-images.githubusercontent.com/210914/46767234-35d4c800-cc99-11e8-9564-b327a8c6a7b3.png)

I'm not convinced this is worth doing though

Reply:
This is cool!